community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
@parth-gohil, the title does not make sense to the contents. This appears to be additional information. Maybe this could be retitled and also moved to a more appropriate position.